client/src/components/modals/DetailsModal.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Modal, Header } from 'semantic-ui-react';
+import 'semantic-ui-css/semantic.min.css';
+
+function DetailsModal({ session }) {
+  const [open, setOpen] = React.useState(false);
+  return (
+    <Modal
+      className="modal-container"
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={<Button className="join-button">JOIN</Button>}
+    >
+      <Modal.Content>
+        <Modal.Description>
+          <Header>Join {session.creator}</Header>
+          <p id="form-text">
+            Class: {session.class}
+            <br />
+            Location: {session.location}
+            <br />
+            Expected Group Size: {session.attendees}
```

Comment:
In the [database](https://docs.google.com/document/d/17DP6zekcG-BVo69tBjTnzAj5OhXQ5b3O8aey9nKG-jE/edit?usp=sharing), `attendees` is a string array. Please replace group size to a list of all attendees. Even though you're currently working with fake data, please make sure the types are consistent with the types in the database to avoid errors in the future once we start using real data!